scripts/install/brew.sh

```
@@ -52,4 +52,12 @@ fi
 
 print_step "Updating Homebrew" && brew update
 print_step "Installing Homebrew packages" && brew install "${packages[@]}"
+
+# repo-intel (standalone tool: github.com/tyom/repo-intel). Installed separately
+# and non-fatally so a missing/unpublished tap can't abort the core packages
+# above. If this is skipped, setup.sh's curl fallback installs it into ~/bin.
+print_step "Installing repo-intel (tap: tyom/homebrew-tap)" &&
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)